composer.json

```diff
@@ -14,7 +14,7 @@
     ],
     "require": {
         "php": ">=5.6.0",
-        "illuminate/support": "5.1.*|5.2.*|5.3.*|5.4.*|5.5.*|5.6.*",
+        "illuminate/support": "5.1.*|5.2.*|5.3.*|5.4.*|5.5.*|5.6.*|5.6.*",
```

Comment:
Why are you using 5.6.* twice here? For laravel 5.7 it should be `"illuminate/support": "5.1.*|5.2.*|5.3.*|5.4.*|5.5.*|5.6.*|5.7.*"`

Reply:
I just realised my error.

Reply:
I will submit a new PR.

Reply:
OK. Thank you!

Reply:
Apologies, about that. Will be creating a new Laravel 5.7.2 project, will let you know if I find any other issues.